etc/db_schema.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="kl_sync" resource="default" engine="innodb" comment="Klaviyo Sync table">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true"
+                comment="Primary key"/>
+        <column xsi:type="text" name="payload" nullable="true" comment="Event payload"/>
+        <column xsi:type="varchar" name="status" nullable="false" length="30" comment="Status of row (NEW, SYNCED...)"/>
+        <column xsi:type="varchar" name="topic" nullable="false" length="255" comment="Klaviyo Topic name"/>
+        <column xsi:type="text" name="klaviyo_id" nullable="true" length="100" comment="Klaviyo ID"/>
```

Comment:
```suggestion
        <column xsi:type="text" name="klaviyo_id" nullable="true" comment="Klaviyo ID"/>
```
When you use a TEXT type for a column you don’t have to specify a storage length like for CHAR and VARCHAR because would hit issue [here](https://github.com/magento/devdocs/issues/7029)